superset/examples/echarts_dashboard.py

```diff
@@ -113,138 +496,813 @@ def load_echarts_dashboard() -> None:
-          "width": 6
-        },
-      "type": "CHART"
-    },
-    "CHART-06Kg-rUggO": {
-      "children": [],
-      "id": "CHART-06Kg-rUggO",
-      "meta": {
-        "chartId": 617,
-        "height": 50,
-        "sliceName": "Number of Girls",
-        "width": 6
-      },
-      "type": "CHART"
-    },
-    "CHART--wEhS-MDSg": {
-      "children": [],
-      "id": "CHART--wEhS-MDS",
-      "meta": {
-        "chartId": 2,
-        "height": 50,
-        "sliceName": "Energy Force Layout",
-        "width": 6
-      },
-      "type": "CHART"
-    },
-    "CHART--LXvS-RDSu": {
-      "children": [],
-      "id": "CHART--LXvS-RDSu",
-      "meta": {
-        "chartId": 398,
-        "height": 50,
-        "sliceName": "Time-Series Bar Chart V2",
-        "width": 6
-      },
-      "type": "CHART"
-    },
-    "GRID_ID": {
-        "children": [
-            "ROW-SytNzNA4X",
-            "ROW-HkFFEzVRVm",
-            "ROW-BytNzNA4Y"
-        ],
-        "id": "GRID_ID",
-        "type": "GRID"
-    },
-    "HEADER_ID": {
-        "id": "HEADER_ID",
-        "meta": {
-            "text": "ECharts Dashboard"
-        },
-        "type": "HEADER"
-    },
-    "ROOT_ID": {
-        "children": [
-            "GRID_ID"
-        ],
-        "id": "ROOT_ID",
-        "type": "ROOT"
+  "CHART-1": {
+    "children": [],
+    "parents": [
+      "ROOT_ID",
+      "TABS-TOP",
+      "TAB-TOP-1",
+      "ROW-1"
+    ],
+    "id": "CHART-1",
+    "meta": {
+      "chartId": 1,
+      "height": 50,
+      "sliceName": "Big Number",
+      "width": 6
     },
-    "ROW-HkFFEzVRVm": {
-        "children": [
-            "CHART-dxV7Il74hH",
-            "CHART-oWKBOJ6Ydh"
-        ],
-        "id": "ROW-HkFFEzVRVm",
-        "meta": {
-            "background": "BACKGROUND_TRANSPARENT"
-        },
-        "type": "ROW"
-    },
-    "ROW-SytNzNA4X": {
-        "children": [
-            "CHART-06Kg-rUggO",
-            "CHART-YyHWQacdcj"
-        ],
-        "id": "ROW-SytNzNA4X",
-        "meta": {
-            "background": "BACKGROUND_TRANSPARENT"
-        },
-        "type": "ROW"
-    },
-    "ROW-BytNzNA4Y": {
-        "children": [
-            "CHART--wEhS-MDSg",
-            "CHART--LXvS-RDSu"
-        ],
-        "id": "ROW-BytNzNA4Y",
-        "meta": {
-            "background": "BACKGROUND_TRANSPARENT"
-        },
-        "type": "ROW"
+    "type": "CHART"
+  },
+  "CHART-2": {
+    "children": [],
+    "parents": [
+      "ROOT_ID",
+      "TABS-TOP",
+      "TAB-TOP-1",
+      "ROW-1"
+    ],
+    "id": "CHART-2",
+    "meta": {
+      "chartId": 2,
+      "height": 50,
+      "sliceName": "Big Number with Trendline",
+      "width": 6
+    },
+    "type": "CHART"
+  },
+  "CHART-3": {
+    "children": [],
+    "parents": [
+      "ROOT_ID",
+      "TABS-TOP",
+      "TAB-TOP-1",
+      "ROW-1"
+    ],
+    "id": "CHART-3",
+    "meta":{
+      "chartId": 3,
+      "height": 50,
+      "sliceName": "Table",
```

Comment:
I'm not seeing this one in the dashboard.

Reply:
They're not missing, I messed something up and the charts overflow into third column 🤦 
<img width="1747" alt="image" src="https://user-images.githubusercontent.com/15073128/192277693-014f5962-9971-49cb-986a-ec76f1c2d18e.png">

Reply:
Lol sorry I should have noticed the pattern!